### PR TITLE
Timeline/Resource Planning

### PR DESCRIPTION
**Pull request by TheTexanCodeur**
This PR aims to add the "Timeline/Resource Planning" part to the PRD

### CODE REVIEW

File: template/10-timeline.md [hunk 1]
```diff
@@ -1,10 +1,92 @@
...
+| Project Manager | 4 |
+
+## Deployment Resources
+
+We use the services of Google to manage our backend and to monitor the app. The map will be self-hosted with OpenStreetMap.
+
+| **Item** | **Cost / unit** | **Units** | **Totals** |
+| --- | --- | --- | --- |
+| OpenStreetMap Hosting | 200 CHF / month | - | 800 CHF |
+| Google Firebase | 150 CHF / month | - | 600 CHF  |
```
Comment (Aderfish) [inline template/10-timeline.md:87]:
```suggestion
| OpenStreetMap Hosting | 200 CHF / month | 4 | 800 CHF |
| Google Firebase | 150 CHF / month | 4 | 600 CHF  |
```

I think you should clarify that we will be using these resources for 4 months here.

File: template/10-timeline.md [hunk 2]
```diff
@@ -1,10 +1,92 @@
...
+- Setup of the backend (Google Firebase)
+- Documentation of backend API
+- Basic frontend building blocks
+- First interation of UI/UX design (Figma) 
+- Google login integration
+- Tests with mock datas 
+
+| **Sprint/Week Number** | **Objective** | **Outcomes** |
+| --- | --- | --- |
+| Week 1 | Preparation of the codebase| Documentation of the backend API. Global view of the project's visual/technical directions. |
```
Comment (Aderfish) [inline template/10-timeline.md:20]: "Documentation of the backend API" what do you mean ?

File: template/10-timeline.md [hunk 3]
```diff
@@ -1,10 +1,92 @@
...
+- Ensure that each new feature is thouroghly tested.
+
+| **Sprint/Week Number** | **Objective** | **Outcomes** |
+| --- | --- | --- |
+| Week 3 | Post creation | Posts with data can be stored and retrieved from the backend. Frontend elements can display this data. |
+| Week 4 | Commenting system | Completed commenting feature in frontend and backend |
+| Week 5 | Voting system | Developped voting system. Unit testing that ensures that database stays consistent |
+| Week 6 | Location-based feed | Combinations of the developped features in the frontend. Mock testing. |
+
+#### Milestone 3
```
Comment (Aderfish) [inline template/10-timeline.md:37]: I suggest you mention the OSM tile server setup somewhere in this milestone as well. That is no small task.

**Review by Aderfish — CHANGES_REQUESTED** [review]
Good job, I left a few small comments.

File: template/10-timeline.md (hunk 3 above)
Comment (TheTexanCodeur) [reply template/10-timeline.md:37]: Fixed with [9344c53](https://github.com/ProximaEPFL/prd/pull/13/commits/9344c5342de58aee99d8e8bf64596d90e89a6fac)

**Review by Aderfish — APPROVED** [review]

**Pull request merged**